### ISSUE TITLE
Strikes Menu improvement proposals

### ISSUE THREAD

**Issue by apoapostolov** [Larkinabout/fvtt-token-action-hud-pf2e#6]
First, rename it from Attacks to Strikes, this is the proper PF2 term.

The menu is a complete mess right now. The default settings for that window and the excessive amount of low-relevance data is eating both horizontal and vertical space, and there's a lot of air between elements that makes players with 6+ weapons (a common sight in the game mid-tiers) to be right unusable. 

The current situation is, without using custom settings to make it more palatable

![image](https://user-images.githubusercontent.com/4083812/218447944-3a8ed37a-a2cb-4c9f-b496-6e4dd897d817.png)

Compare this with old TAH PF2 with Dorako UI

![image](https://user-images.githubusercontent.com/4083812/218448131-c776c16a-8895-4a1f-a802-df9258fb9144.png)

Here are the major issues:

**Comment by apoapostolov** [comment]
I just noticed that I can remove titles to subsections and even single weapons (though I suspect not practical because weapons can be changed and edited, renamed, etc.) but still it doesn't cover things like reordering elements back to be more compact and practical from 2 lines (secondary first, primary second) to old optimal way (icon, primary first, secondary second)

**Comment by Larkinabout** [comment]
This will probably improve with options to move subheadings around as well as hide them. I'm also working on this feature: Larkinabout/fvtt-token-action-hud-core/issues#19 to allow any action to be added to any subcategory, meaning you could add the Sheathe/Draw/Drop/Change Grip actions to the same subcategory at the attack/damage actions.

Coming as a new player/GM to Pathfinder 2 (thanks to Abomination Vaults on Humble Bundle), I actually love the separate lines for as it feels like I'm actually drawing those weapons, in a weird sense, when those subgroups appear underneath.

There's another feature issue about hiding titles by default, so that'll save you some more space in the long run: Larkinabout/fvtt-token-action-hud-pf2e#3

**Comment by Larkinabout** [comment]
Also, the default 'Actions' name is based on the character sheet tab, which groups Strikes, Actions, Reactions and Free Actions.

**Comment by apoapostolov** [comment]
I agree with you on the importance of drawing and sheathing, as well as dropping (also running AV btw). But it doesn't need to be on two lines. If a weapon si eathed, it will not display its MAP and Damage buttons thus be the same line. If the weapon is drawn, then it would display them, followed by the Sheathe and Drop buttons. This is how it worked before.

**Comment by apoapostolov** [comment]
> Also, the default 'Actions' name is based on the character sheet tab, which groups Strikes, Actions, Reactions and Free Actions.

Understood, and very easy to customize back to old TAH Classic way so I only proposed that because I thought the Actions section was only Strikes as before. Again, very easy to customize now. The only real issue is the strike format.  

Oh and the overuse of 1A icons in MAPs and Damage, should I open another issue for this?

**Comment by Larkinabout** [comment]
> > Also, the default 'Actions' name is based on the character sheet tab, which groups Strikes, Actions, Reactions and Free Actions.
> 
> Understood, and very easy to customize back to old TAH Classic way so I only proposed that because I thought the Actions section was only Strikes as before. Again, very easy to customize now. The only real issue is the strike format.
> 
> Oh and the overuse of 1A icons in MAPs and Damage, should I open another issue for this?

Yeah, on Damage and Critical, I'm not sure why they're there, but on the attack actions those are technically correct as indicators of action usage.

**Comment by apoapostolov** [comment]
> Yeah, on Damage and Critical, I'm not sure why they're there, but on the attack actions those are technically correct as indicators of action usage.

You're right but the problem is that the icons are visually overwhelming when repeated on every button in that format. From UX point they are not indicative for exception (only default) while they draw attention away from the value (which is the most important information). Since 1A to strike is absolute essential information taught the moment the DM starts running combat, it doesn't communicate any information that needs to be learned or referenced.

**Comment by JDCalvert** [comment]
Can I tack onto this that it would be great if Token Action HUD respected the option to not show the basic unarmed attack?
